Adyen/UI/Form/Items/Basic Items/FormLabelItem.swift

```diff
@@ -21,24 +22,35 @@ public class FormLabelItem: FormItem {
         self.text = text
     }
 
+    public init(
+        text: String,
+        identifier: String? = nil,
+        labelStyle: LabelStyle = LabelStyle()
```

Comment:
Shall we consider to make it just `style` as "label" context is provided by its type for free?